plugins/linuxplugin/ifplugin/interface_config.go

```diff
@@ -941,3 +986,41 @@ func (plugin *LinuxInterfaceConfigurator) prepareVethConfigNamespace() error {
 	plugin.vethCfgNamespace, err = linuxcalls.ToInterfaceNs(ns)
 	return err
 }
+
+func (plugin *LinuxInterfaceConfigurator) vethExists(iface string, max int) bool {
+	for attempt := 1; attempt <= max; attempt++ {
+		_, err := net.InterfaceByName(iface)
+		if attempt == max {
```

Comment:
This part is unnecessary since after reaching max attempts the for loop exits and same message will be logged and same value will be returned.